packages/@react-spectrum/breadcrumbs/src/Breadcrumbs.tsx

```diff
@@ -25,6 +25,42 @@ import {useProviderProps} from '@react-spectrum/provider';
 const MIN_VISIBLE_ITEMS = 1;
 const MAX_VISIBLE_ITEMS = 4;
 
+type useResizeObserverOptionsType<T> = {
+  ref: RefObject<T>,
+  onResize: (size: {
+    width: number,
+    height: number,
+  }) => void
+}
+
+function useResizeObserver<T extends HTMLElement>(options: useResizeObserverOptionsType<T>) {
+  const {ref, onResize} = options;
+
+  const resizeObserverInstance = new ResizeObserver((entries) => {
```

Comment:
can we move the instantiation of the resize observer inside the useEffect? otherwise we're creating a bunch that don't use

Reply:
should we also fall back to window resize events when ResizeObserver isn't available (e.g. old edge/safari)?